portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,46 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+// function addRandomGreeting() {
+//   const greetings =
+//       ["What we know is a drop and what we don't know is an ocean.", "Beets, Bears, Battlestar Galactica", "To live is to risk it all.", "People don't change. They become more of who they are."];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+//   // Pick a random greeting.
+//   const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+
+//   // Add it to the page.
+//   console.log("this is a function for random quotes");
+//   const greetingContainer = document.getElementById('greeting-container');
+//   greetingContainer.innerText = greeting;
+// }
+
+function addProjectDetail(buttonId) {
+  const description =
```

Comment:
Since it is an array, `descriptions` seems to be a better name.

Reply:
Sure